lib/events/index.js

```diff
@@ -546,6 +554,11 @@ export default function (self) {
             }
           } else if (dragBounds.top !== -1) {
             self.selectArea(sBounds, true);
+            if (
+              sBounds.left == -1 &&
+              Math.abs(sBounds.top - sBounds.bottom) > 0
```

Comment:
Not sure I completely understand what's going on here: `Math.abs` will turn `sBounds.top - sBounds.bottom` into a positive number if it's negative, so this expression will either be a positive number or zero; so I believe this expression is equivalent to checking whether `sBounds.top` and `sBounds.bottom` are different? As in:

```js
Math.abs(sBounds.top - sBounds.bottom) > 0
// is equivalent to:
sBounds.top !== sBounds.bottom
```

Is that correct? If so, I think this is maybe clearer? What do you think?

Reply:
yes, that is correct